Assignment-16/AdvancedDelegates/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace AdvancedDelegates
+{
+    internal class Program
+    {
+     public delegate int SortDelegate(Product firstProduct,Product secondProduct);
```

Comment:
```suggestion
     public delegate int Sorter(Product firstProduct, Product secondProduct);
```
Give proper spacings